apps/web/components/pages/eventtypes/CustomInputTypeForm.tsx

```diff
@@ -73,7 +73,7 @@ const CustomInputTypeForm: FC<Props> = (props) => {
             type="text"
             id="label"
             required
-            className="focus:border-primary-500 focus:ring-primary-500 block w-full rounded-sm border-gray-300 shadow-sm sm:text-sm"
```

Comment:
no idea why we have introduced bigger text in input boxes on mobile. just looks odd. removed it and gave everything "text-sm"